### DISCUSSION TITLE
Wobbling stars in ICRS grid

### DISCUSSION THREAD

**Discussion by diegorodriguezperegrin** [Stellarium/stellarium#3375] [General]
Hello. Maybe I am confused on the meaning of the ICRS grid, but shouldm't it be free of star aberration? I understand ICRS was meant to be Earth independent, including precession, nutation, aberration and any other date dependent equinox changes. Many thanks!

**Comment by gzotti** [comment]
Probably we should rename our ICRS grid to J2000.0 grid then to avoid confusion. We also label our J2000.0 coordinates as such.  @alex-w ?

However, as I understand, the grid is defined by remote quasars and should be fixed as such. The observable stars will of course be affected by all those effects. ICRS coordinates are mean catalog values, from which observable positions are computed.

**Reply by diegorodriguezperegrin** [reply]
If you select an ICRS grid, stars will not be wobbling because of by nutation, precession or any other Earth orientation change. Computing topocentric values from ICRS coordinates can be done for instance using http://www.iausofa.org/2021_0512_C/sofa/atoc13.c

**Reply by gzotti** [reply]
We don't use SOFA but other sources which should provide comparable results. SOFA should have been around earlier...

To get as close as possible to "certified" ICRS coordinates, switch off aberration simulation and set date to J2000.0 to avoid correction for stellar proper motion. And please do not expect milliarcsecond accuracy.

**Reply by alex-w** [reply]
> Probably we should rename our ICRS grid to J2000.0 grid then to avoid confusion. We also label our J2000.0 coordinates as such. @alex-w ?

We have "ICRS" text onto the icon only
